### PR TITLE
feat(prismic-api): call api without follow interceptor chain

### PR DESCRIPTION
**Pull request by heavybeard**
Close https://github.com/exportarts/ngx-prismic/issues/23

### CODE REVIEW

File: lib/prismic-client/src/services/prismic.service.ts [hunk 1]
```diff
@@ -38,12 +38,14 @@ export const API_TOKEN = new InjectionToken<string>('API_TOKEN');
   providedIn: 'root'
 })
 export class PrismicService {
+  private http: HttpClient;
```
Comment (evolkmann) [inline lib/prismic-client/src/services/prismic.service.ts]: Can be readonly

File: lib/prismic-client/src/services/prismic.service.ts [hunk 2]
```diff
@@ -38,12 +38,14 @@ export const API_TOKEN = new InjectionToken<string>('API_TOKEN');
...
 
   constructor(
-    private readonly http: HttpClient,
+    private httpBackend: HttpBackend,
```
Comment (evolkmann) [inline lib/prismic-client/src/services/prismic.service.ts]: Can be readonly

File: lib/prismic-client/src/services/prismic.service.ts (hunk 1 above)
Comment (heavybeard) [reply lib/prismic-client/src/services/prismic.service.ts]: changed

File: lib/prismic-client/src/services/prismic.service.ts (hunk 2 above)
Comment (heavybeard) [reply lib/prismic-client/src/services/prismic.service.ts]: changed

**Comment by evolkmann** [comment]
Hi @heavybeard, your PR has been merged and released in [v1.8.1](https://www.npmjs.com/package/ngx-prismic/v/1.8.1).

**Pull request merged**